src/tools/rbd_mirror/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ add_library(rbd_mirror_types STATIC
 set(rbd_mirror_internal
   ClusterWatcher.cc
   ImageDeleter.cc
+  ImageMapper.cc
```

Comment:
The problem is that the Replayer needs to track in some way what images have already been acquired. It needs this to make a decision to release an image that was deleted in the peer pool. I expect it will be getting this info from ImageMapper in future, thus an extremely simplified version has been added.

I could just add a temporary ImageMapper class (or whatever it is called) to Replayer. Or add a method to InstanceReplayer to get e.g. a list of acquired images and use this instead, but I suppose this is not a way we a going to do this eventually so it would also be a temporary solution?